### PR TITLE
Update Node feature test scripts

### LINKED ISSUE #725: Testcase for hedera services update of N node network to N+1 Nodes
**Summary**
We need a new test case to test the special services update scenario: while doing services update, there are new nodes added to the services network. 

Simply put, during update, the config.txt will also be updated with added node(s) to the existing nodes, the update should still proceed as normal.

**Test Steps**

1. N node network @version v0.X.0
2. N+1th node prepped with v0.X.1 and new config.txt 
3. Build Update ZIP to include v0.X.1 
4. Build Update ZIP to include new config.txt with N+1 nodes with steps to copy
5. Pause All nodes for the duration of State copy to node N+1 
6. Use Update client to send update ZIP to N nodes of network
7. Use Update client to send freeze with 0.0.150 fileID. (This will cause freeze and update code and new config.txt)
8. Copy latest freeze State (*.swh &PGbackup) from Node 0 to node N+1
9. Start JAR on N+1th node
10. All N+1 nodes should (re)start, running v0.X.1 with last freeze state.
11. All N+1 Nodes should go gossip and accept and handle transactions. 

**Implementation:**
- [x] Add `useUpdateNode` (for now the value is 1 ) json config to HederaServicesConfig
- [x] Instantiate all the nodes as usual , as mentioned in cloudConfig 
Could Config should be as follows
```
"cloud": {
	"regionList": [
	  {
		"region": "us-east-2",
		"numberOfNodes" : 5,
		"numberOfTestClientNodes": 1
	  }
	]
  }
```
- [x] For the update feature if `useUpdateNode` is present treat the last staked node as update node
- [x] send all files to all the nodes
- [x] Initially while generating `config.txt` generate only for the `totalNodes - updateNodes` number of addresses
- [x] Don't start Browser on the last staked node
- [x] Generate a new config.txt on `hedera-services/test-clients/updateFiles` path with the newly added node
- [x] After update Transfer the newly generated config.txt and the saved state files to 5th node
- [x] Start 5th node in addition to other 4 nodes, and run updateServerTest
- [x] Validate 5th node is going to MAINTENANCE similar to other nodes
- [x] Modify HGCAAQuery validator to add new validations

Being implemented in https://github.com/swirlds/swirlds-platform-regression/compare/Services-725-D-Update-Node-Config

### PR DESCRIPTION
**Pull request by Neeharika-Sompalli**
**Related issue(s)**:
Closes #725 

**Summary of the change**:
Added test scripts that are called by platform JRS while testing update node feature.

**External impacts**:
None.

**Applicable documentation**
Passed regression test : https://hedera-hashgraph.slack.com/archives/C018CBQBTGB/p1605077353000100
https://hedera-hashgraph.slack.com/archives/C018CBQBTGB/p1605079503000500

### CODE REVIEW

File: test-clients/scripts/updateNodeTransfer.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+#
+#
+set -eE
+UPDATE_NODE_IP_ADDRESS=""
+NODE0_IP_ADDRESS=""
+CONFIG_FILE_PATH=""
+copyFilesToUpdateNode() {
+  # copy config.txt , File0.0.150 and saved state to update nodes to start from the state
+  set -x
+  sudo scp -o StrictHostKeyChecking=no -i services-regression.pem -p $CONFIG_FILE_PATH ubuntu@$UPDATE_NODE_IP_ADDRESS:/home/ubuntu/remoteExperiment
```
Comment (ljianghedera) [inline test-clients/scripts/updateNodeTransfer.sh:13]: I think without 'sudo', this command should work too.

File: test-clients/scripts/regressionFlowUpdateFiles.sh [hunk 2]
```diff
@@ -61,4 +61,26 @@ updateServiceMainJava()
 
 }
 
-updateServiceMainJava
+PLATFORM_REGRESSION_REPO_PATH=""
```
Comment (anighanta) [inline test-clients/scripts/regressionFlowUpdateFiles.sh:64]: I this used somewhere ?

**Review by anighanta — COMMENTED** [review]
LGTM

**Review by ljianghedera — DISMISSED** [review]
Overall LGTM.

File: test-clients/scripts/regressionFlowUpdateFiles.sh (hunk 2 above)
Comment (Neeharika-Sompalli) [reply test-clients/scripts/regressionFlowUpdateFiles.sh:64]: Yes it is used in line 71

Comment (anighanta) [reply test-clients/scripts/regressionFlowUpdateFiles.sh:64]: `cp "$PLATFORM_REGRESSION_PATH/tmp/config.txt" $TARGET_DIR` ??

Comment (Neeharika-Sompalli) [reply test-clients/scripts/regressionFlowUpdateFiles.sh:64]: Oh you are correct. It is not used anywhere. Will remove it. Thank you!!

**Review by anighanta — APPROVED** [review]

File: test-clients/scripts/updateNodeTransfer.sh (hunk 1 above)
Comment (Neeharika-Sompalli) [reply test-clients/scripts/updateNodeTransfer.sh]: Agreed. removed it. Thanks @ljianghedera

**Review by ljianghedera — APPROVED** [review]
LGTM

**Review by JeffreyDallas — APPROVED** [review]

**Review by ljianghedera — APPROVED** [review]
LGTM

**Pull request merged**